src/components/common/QR.tsx

```diff
@@ -59,6 +59,7 @@ export default function QR(props: QRProps) {
           justifyContent: 'center',
           alignItems: 'center'
         }}
+        data-challenge={props.value}
```

Comment:
why is this changed?

Reply:
We need to log in, and to log in we need to get the `challenge` in Cypress.

This is to enable us to perform the log in function the app does.